src/workspaces/types.ts

```diff
@@ -18,6 +19,7 @@ export type Vocabulary = Omit<
   'types' | 'basedOnVocabularyVersion' | 'changeTrackingContext'
 > & {
   vocabulary: Uri
+  label?: string
```

Comment:
The label definition here is extra as it is already contained in VocabularyData. With that said - why is the label optional? If we want to make it required, which we probably are, then it makes no sense to have it specified as optional in here.

Reply:
1) ".. label definition here is extra .."
I will fix it.

2) " .. If we want to make it required, which we probably are, then it makes no sense to have it specified as optional in here. .."
Well, I agree, but I think it is rather "required in the future" now as there is a lot to discuss how it should work. The label is taken (inferred) from a vocabulary context. Thus if you add vocabulary from the control-panel now, it will not have any label anyway.   Thus should control-panel create vocabulary context as well?

Reply:
The app works in a way that it creates the vocabulary and then re-fetches the list of vocabularies for a workspace. So if the context is created on backend, it should be immediately visible to the user as well. But fair enough regarding the label, let's keep that optional.